src/management_console/views/pollers/EditPoller.js

```diff
@@ -150,7 +150,15 @@ export default class EditPoller extends Component {
   savePoller() {
     this.disable();
     if (this.state.poller.id) {
-      this.pollers.update(this.state.poller.id, this.state.poller).then(() => this.enable());
+      let poller = this.state.poller;
+      let updateData = {
+          type : poller.type,
+          pollInterval: poller.pollInterval,
+          node : poller.node,
+          config: poller.config,
+          paused: poller.paused
+      };
+      this.pollers.update(this.state.poller.id, updateData).then(() => this.enable());
```

Comment:
Suggestion: can pass function reference into then() directly, instead of anonymous function. 
e.g. XXX().then(()=>this.enable())   ==>    XXX().then(this.enable);

Reply:
Great. That will be more clear.
`this.pollers.update(this.state.poller.id, updateData).then(this.enable);  `